src/Services/FlowConfig/FlowConfig.php

```diff
@@ -0,0 +1,72 @@
+<?php
+declare(strict_types=1);
+
+namespace LoyaltyCorp\Multitenancy\Services\FlowConfig;
+
+use CodeFoundation\FlowConfig\Interfaces\ConfigRepositoryInterface;
+use CodeFoundation\FlowConfig\Interfaces\EntityConfigRepositoryInterface;
+use LoyaltyCorp\Multitenancy\Services\FlowConfig\Interfaces\FlowConfigInterface;
+use LoyaltyCorp\Multitenancy\Services\FlowConfig\Interfaces\FlowConfigurableInterface;
+
+final class FlowConfig implements FlowConfigInterface
+{
+    /**
+     * @var \CodeFoundation\FlowConfig\Interfaces\EntityConfigRepositoryInterface
+     */
+    private $entityFlowConfig;
+
+    /**
+     * @var \CodeFoundation\FlowConfig\Interfaces\ConfigRepositoryInterface
+     */
+    private $flowConfig;
+
+    /**
+     * FlowConfig constructor.
+     *
+     * @param \CodeFoundation\FlowConfig\Interfaces\EntityConfigRepositoryInterface $entityFlowConfig
+     * @param \CodeFoundation\FlowConfig\Interfaces\ConfigRepositoryInterface $flowConfig
+     * @param bool|null $autoFlush
```

Comment:
Good work on seeing that we'll need this. I recommend either adding a note in the code to reference the ticket for autoflush, or a reference to this un-used argument in the ticket.